RecentChangesWebhooksJob.php

```diff
@@ -0,0 +1,14 @@
+<?php
+class RecentChangesWebhooksJob extends Job {
+	static function Of($change) {

```

Comment:
probably typo - called as `::of` (`RecentChangesWebhooks.class.php:+29`) but defined as `::Of`
```suggestion
	static function of($change) {
```